src/services/dashboards/dashboard-customize/modules/VariableMoreButtonDropdown.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="variable-more-button">
+        <p-button icon-left="ic_plus"
+                  style-type="highlight"
+                  @click="handleClickButton"
+        >
+            {{ $t('DASHBOARDS.CUSTOMIZE.VARIABLES.MORE') }}
+        </p-button>
+        <p-context-menu v-if="state.visibleMenu"
+                        :menu="state.variables"
+                        :selected="state.selected"
+                        multi-selectable
+                        searchable
+                        use-fixed-menu-style
+                        show-clear-selection
+                        @select="handleVariableSelect"
+        >
+            <template #bottom>
+                <p-button style-type="secondary"
+                          icon-left="ic_setting"
+                          @click="handleOpenOverlay"
+                >
+                    {{ $t('DASHBOARDS.CUSTOMIZE.VARIABLES.TITLE') }}
+                </p-button>
+            </template>
+        </p-context-menu>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import {
+    computed, onMounted, reactive,
+} from 'vue';
+
+import { PButton, PContextMenu } from '@spaceone/design-system';
+import type { MenuItem } from '@spaceone/design-system/dist/src/inputs/context-menu/type';
+
+import { SpaceRouter } from '@/router';
+
+import type { DashboardVariableSchemaProperty } from '@/services/dashboards/config';
+import { MANAGE_VARIABLES_HASH_NAME } from '@/services/dashboards/config';
+
+interface Props {
+    variableMap: {[key: string]: DashboardVariableSchemaProperty };
```

Comment:
suggestion: I think it'd be better to make it a type. like `VariableProperty`? `VariableMap`?